### ISSUE TITLE
Reliable crash when playing Designed for Danger part 4

### ISSUE THREAD

**Issue by jacopofar** [ValveSoftware/portal2#365]
Hello and thank you a lot for the attention to Linux as a platform for games!

In my case Portal 2 works without issues and so do most community test chambers, but "Designed for danger part 4" crashes Portal 2 every time as soon as I enter the very first room. I can reproduce the crash every time, and it seems it happens when the player crosses a specific area at the beginning of the test, skipping it with a portal doesn't trigger it.

I'm on Fedora 34, with Wayland as per default using AMD integrated graphic, using the Flatpak Steam client.
The computer is Thinkpad P14s with an AMD Ryzen 7 and no dedicated GPU
Here is my system information from the Steam client:
https://gist.github.com/jacopofar/0046b3e671eb2adee4f5f2f0c785b04c

**Comment by misyltoad** [comment]
Can you please link the test chamber?

**Comment by kisak-valve** [comment]
https://steamcommunity.com/sharedfiles/filedetails/?id=104609527 is the first workshop hit.

**Comment by jacopofar** [comment]
Yes, it's https://steamcommunity.com/sharedfiles/filedetails/?id=104609527

**Comment by misyltoad** [comment]
Thanks, this will be fixed in the next update!
Thanks for reporting!
